src/components/Editor.vue

```diff
@@ -761,6 +761,22 @@ export default {
 			console.debug(editor.getHTML())
 		},
 
+		/**
+		 * Helper method to debug yjs issues
+		 */
+		debugYjs() {
```

Comment:
Do we have this somewhat accessible when text open in viewer so we can also ask people editing to dump the output when editing outside of collectives?

Reply:
Having an array of editors sounds reasonable. We can add the filename maybe to the debug output so that the individual ones can be identified (e.g. a readme is open and a viewer instance).

I think we don't need to make it too complex, as long as we can somehow get the information and identify the relevant instance.

Reply:
`OCA.Text.debugYjs` now prints the debugging data of all registered editor instances along with instructions, see updated screenshot above.